CodeGen/kubernetes/codegen_gaudi.yaml

```diff
@@ -29,6 +29,6 @@ spec:
         internalService:
           serviceName: tgi-gaudi-svc
           config:
-            MODEL_ID: meta-llama/CodeLlama-7b-hf
+            MODEL_ID: ise-uiuc/Magicoder-S-DS-6.7B
```

Comment:
hi @zhlsunshine  "ise-uiuc/Magicoder-S-DS-6.7B" does not works under tgi2.2.0 refer to https://github.com/opea-project/GenAIExamples/issues/625

Reply:
Okay, so let's keep using `meta-llama/CodeLlama-7b-hf` for gaudi.